tests/assembler.rs

```diff
@@ -1,6 +1,8 @@
 // SPDX-License-Identifier: (Apache-2.0 OR MIT)
 // Copyright 2017 Rich Lane <lanerl@gmail.com>
 
+// These tests should only be run on std as they depend on the assembler.
+#![cfg(feature = "std")]
```

Comment:
I'm confused here as well, can't we enable some of these tests now?

Reply:
I think those were reenabled in one of the subsequent commits. I decided treat enabling no_std in the assembler and disassembler (and enabling their corresponding tests) as a separate feature as it required changing the way we use `combine` library. I can squash the two commits together so that it is no longer confusing

Reply:
Oh I doublechecked, sorry about this. Those can indeed be enabled without issues